src/tbp/monty/frameworks/experiments/monty_experiment.py

```diff
@@ -51,19 +51,25 @@ class MontyExperiment:
     the outermost loops for training and evaluating (including run epoch and episode)
     """
 
-    def setup_experiment(self, config):
-        """Set up the basic elements of a Monty experiment and initialize counters.
+    def __init__(self, config):
+        """Initialize the experiment based on the provide configuration.
 
         Args:
             config: config specifying variables of the experiment.
         """
-        # Save a copy of the config used to specify the experiment before modifying
+        # Copy the config and store it so we can modify it freely
         config = copy.deepcopy(config)
-        # Convert any dataclass back to dict for backward compatibility
```

Comment:
Just curious why you removed this comment. Is it not accurate?

Reply:
I thought it was redundant because it's basically summarizing the docstring of `config_to_dict`. 

The other comment above got rewritten because I originally copied the code up from `setup_experiment` without the comment and wrote a new one. I didn't realize I ended up just rewriting the original comment until I saw the diff. I think my version _might_ be explaining the "why" of the copying better than the original, but I could just as easily keep the original comment. 🤷

Reply:
okay yeah makes sense. I'm happy with how it is. I was just curious if something changed in the functionality that I missed.